src/x/config/config.go

```diff
@@ -42,37 +41,50 @@ var errNoFilesToLoad = errors.New("attempt to load config with no files")
 type Options struct {
 	DisableUnmarshalStrict bool
 	DisableValidate        bool
+
+	// LookupFunc -- see go.uber.org/config.Expand
+	Expand config.LookupFunc
```

Comment:
hmm - you might need to set your default relative commit since this is in #2016

Reply:
Whoops yeah sorry about that.